nifi-docs/src/main/asciidoc/administration-guide.adoc

```diff
@@ -2074,7 +2074,25 @@ The Provenance Repository contains the information related to Data Provenance. T
 
 |====
 |*Property*|*Description*
-|nifi.provenance.repository.implementation|The Provenance Repository implementation. The default value is org.apache.nifi.provenance.PersistentProvenanceRepository and should only be changed with caution. To store provenance events in memory instead of on disk (at the risk of data loss in the event of power/machine failure), set this property to org.apache.nifi.provenance.VolatileProvenanceRepository.
+|nifi.provenance.repository.implementation|The Provenance Repository implementation. The default value is org.apache.nifi.provenance.PersistentProvenanceRepository.
+Two additional repositories are available as and should only be changed with caution.
```

Comment:
I agree - that was there previously when the only two options were Volatile and Persistent Prov Repo and the note was there to warn that you should know what you're doing when you change to Volatile. This warning can be removed now, I think, since there are two repos that provide persistent storage of the data.